app/controllers/taxes_controller.rb

```diff
@@ -0,0 +1,79 @@
+class TaxesController < ApplicationController
+  before_action :set_car
+  before_action :set_tax, only: %i[ show edit update destroy ]
+
+  # GET /taxes or /taxes.json
+  def index
+    @taxes = @car.taxes
+    #@taxes = Tax.all
+  end
+
+  # GET /taxes/1 or /taxes/1.json
+  def show
```

Comment:
empty actions could be just removed or use preferable syntax for it like `def show; end`